vllm/entrypoints/utils.py

```diff
@@ -324,3 +324,8 @@ def log_non_default_args(args: Union[argparse.Namespace, EngineArgs]):
         "Must be argparse.Namespace or EngineArgs instance.")
 
     logger.info("non-default args: %s", non_default_args)
+
+
+def should_include_usage(stream_options_include_usage: Optional[bool],
+                         enable_force_include_usage: bool) -> bool:
+    return stream_options_include_usage or enable_force_include_usage
```

Comment:
I was actually hoping to be able to group a bit more of the logic not just wrapping OR tbh.
Going by heart, but what about the block 
```
        if stream_options:
            include_usage = stream_options.include_usage \
                            or enable_force_include_usage
            include_continuous_usage = include_usage and \
                                       stream_options.continuous_usage_stats
        else:
            include_usage, include_continuous_usage = False, False
```
If this is not possible then please revert.